### DISCUSSION TITLE
Clean batch list of files from all metadata?

### DISCUSSION THREAD

**Discussion by slrslr** [mifi/lossless-cut#1817] [Ideas]
Hello,

I suspect that some apps or websites insert sensitive metadata into the video files and when you share these files it may be privacy concern. Is it reasonable to expect that some metadata cleanup function would solve that issue or not? In LosslessCut, I have MOV/MP4 metadata preservation disabled on Export dialog, i am unsure to which extent this is effective and how much sensitive data may remain in the exported file. Here I am rather talking about removing metadata from lets say folder or multiple folders worth of many video files all at once.

**Comment by mifi** [comment]
Not sure which metadata you’re talking about. But I don’t think it’s possible to safeguard against secret data planted into a video. I know it’s possible to embed an invisible signature into an image into its pixels, invisible to the eye, only visible to those who know the algorithm, so even if re-encoding, this secret data willl still be present. Used for example to track pirated movies. I don’t think losslesscut can do a very good job at removing secret data. unless you’re talking aboyt some specifics then I’m not sure what can be done.
